src/contracts/libraries/BeaconChainProofs.sol

```diff
@@ -289,7 +289,7 @@ library BeaconChainProofs {
     }
 
     /// @dev Retrieves a validator's effective balance (in gwei)
-    function getEffectiveBalanceGwei(bytes32[] memory validatorFields) internal pure returns (uint64) {
+    function getEffectiveBalanceGwei(bytes32[] memory validatorFields) public pure returns (uint64) {
```

Comment:
Why was this changed to public? We probably don't want that -- this will resolve to a `delegatecall`, pretty sure.

Reply:
Ahh this was needed for a test but I must have mistaken it for being in the BeaconChainMock file

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/591/commits/20bf8100b15a60ae01f3319eac1a97ebea1a13b7